app/main/helpers/search_helpers.py

```diff
@@ -7,7 +7,12 @@
 
 
 def get_lot_from_request(request, all_lots):
-    lot = request.args.get('lot', None)
+    # TODO prefer get_lot_from_args - get rid of this
```

Comment:
Given this is a simple function, it feels like it would be nice to just rip out the 'old' one rather than put in a TODO that will be quickly ignored.

Reply:
Agree, I was just in a rush. I can still do that before this lands :)